Tests/chk_adrs.py

```diff
@@ -18,7 +18,8 @@
 # | `type`          | REQUIRED                                                                   | one of `Procedural`, `Standard`, or `Decision Record`                                 |
 # | `status`        | REQUIRED                                                                   | one of `Proposal`, `Draft`, `Stable`, `Deprecated`, or `Rejected`                     |
 # | `track`         | REQUIRED                                                                   | one of `Global`, `IaaS`, `KaaS`, `IAM`, `Ops`                                         |
-# | `obsoleted_at`  | REQUIRED if `status` is `Deprecated`                                       | ISO formatted date indicating the date after which the deprecation is in effect       |
+# | `deprecated_at` | REQUIRED if `status` is `Deprecated`                                       | ISO formatted date indicating the date after which the deprecation is in effect       |
+# | `obsoleted_at`  | REQUIRED if `status` is `Obsoleted`                                        | ISO formatted date indicating the date after which the obsoletion is in effect        |
```

Comment:
```suggestion
```

Reply:
Thx for the catch, I thought I fixed that (or better I'm pretty sure about it), but somehow the changes ended up in limbo.